.github/workflows/nightfalldlp.yml

```diff
@@ -0,0 +1,22 @@
+name: nightfalldlp
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  nightfalldlp:
+    name: nightfalldlp
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo Action
+        uses: actions/checkout@v2
```

Comment:
Will customers be required to have this step in the workflow containing our action?

Reply:
yes